### ISSUE TITLE
Suggestion: Get player's currently active job and average item level

### ISSUE THREAD

**Issue by Bwin4L** [XIVStats/XIVStats#8]
It could be a good idea to grab the player's currently active job and their average item level (and, why not, the main weapon's item level too) from the player lodestone profile along with the rest of the currently scraped data. That would allow us to get a "snapshot" of what jobs are players playing at the time of the census, and maybe infer their main job from this additional data (e.g. by comparing levels or previous censuses). And it could also help us with distinguishing SCHs from SMNs.

Could be reflected in the DB with a column containing the class/job trigram of the currently active class in the lodestone, and another containing the average ilvl.

**Comment by AdriaanDeVos** [comment]
Good call, but it would allow for some really fuzzy data because people also log out on alt-classes and crafting/gather classes. This would in my opinion only work for really large datasets to get a general line. And you then need to detach the battle/craft/gather classes from each other for a job breakthrough graph.

Not saying that it wouldn't work. This should definitely be tried out.

**Comment by Bwin4L** [comment]
That's right, it would definitely need careful analysis before exploitation.
I also believe that getting that data (active job, ilevel, main weapon name/ilevel and player HP) could certainly help in determining activity.
I've performed some tests on my side by selecting all ids whose class levels or mounts or minions changed between the february and march censuses, and I'm getting around 560k "active" players.
Unfortunately, that info isn't sufficient because we're not getting players that have their jobs at max level and who aren't actively leveling other classes (but could be doing other content such as anima weapon grinding, weeklies or EX content).
Adding the aforementioned info would help us determine who has been active in the last month by simply checking for changes in these columns.
